components/01-network/11-inputs-default.tf

```
@@ -15,6 +15,8 @@ variable "private_endpoint_private_dns_zones" {
     "privatelink.postgres.database.azure.com",
     "privatelink.dev.azuresynapse.net",
     "privatelink.sql.azuresynapse.net",
+    "privatelink.redis.windows.net",
+    "privatelink.service.signalr.net"
```

add trailing comma to reduce diff
```suggestion
    "privatelink.service.signalr.net",
```